src/components/Hero/_Hero.scss

```diff
@@ -67,6 +67,39 @@
   }
 }
 
+.hero--tertiary {
+  background-color: var(--ui-gray-xdark);
+
+  .nypl--books-and-more & {
+    background-color: var(--section-books-and-more-primary);
+  }
+
+  .nypl--locations & {
+    background-color: var(--section-locations-primary);
+  }
+
+  .nypl--research & {
+    background-color: var(--section-research-primary);
+  }
+
+  .nypl--whats-on & {
+    background-color: var(--section-whats-on-primary);
+  }
+
+  .hero__content {
+    @include wrapper;
+
+    color: var(--ui-white);
+    display: flex;
+    flex-flow: column nowrap;
+    padding: var(--space-m) var(--space);
+
+    .hero__heading {
+      @include space-stack-none;
```

Comment:
Based on the Figma, the spacing between the header and the body text should be increased to `space-stack-s`.